client/src/components/BedroomsCounter.jsx

```diff
@@ -1,20 +1,35 @@
 import React from 'React';
 
 const BedroomsCounter = (props) => {
+  if (props.bedMin === 1) {
```

Comment:
why not use state to keep track of this? or use turnery directly in your jsx

Reply:
great idea thanks. I will make that change.